www/components/project-select.tsx

```diff
@@ -81,8 +81,8 @@ const projects = [
   {
     title: "Effection",
     description: "Structured Concurrency for JavaScript",
-    url: "/V2/",
-    version: "v2",
+    url: "/",
+    version: "v3",
```

Comment:
This was so that you could switch from v3 to v2. There's no need to switch back to v3 if you're on v3 is there?

Is it confusing? If so, maybe we take it out of the switcher altogether?